RIGS/templates/RIGS/eventauthorisation_request.html

```diff
@@ -0,0 +1,32 @@
+{% extends request.is_ajax|yesno:'base_ajax.html,base.html' %}
+{% load widget_tweaks %}
+
+{% block title %}Request Authorisation{% endblock %}
+
+{% block content %}
+    <div class="row">
+        <div class="col-sm-offset-1 col-sm-10">
+            <form action="{{ form.action|default:request.path }}" method="post" class="form-horizontal">{% csrf_token %}
+
+                <div class="row">
+                    {% include 'form_errors.html' %}
+
+                    <div class="form-group">
+                        <label class="col-sm-2 control-label"
+                               for="{{ form.email.id_for_label }}">{{ form.email.label }}</label>
```

Comment:
This popup needs more warnings.

YOU ARE ABOUT TO SEND THIS TO A CLIENT, QUADRUPLE CHECK ALL THE THINGS, DON'T BE A MORON

The client will be able to see all contact info, event items, the event title, the event description (not notes), and the MIC name.